part1-장재혁-week5/src/signup.js

```diff
@@ -0,0 +1,105 @@
+const emailInput = document.getElementById('emailInput');
+const passwordInput = document.getElementById('passwordInput');
+const passwordRepeatInput = document.getElementById('passwordRepeatInput');
+const errorEmail = document.getElementById('errorEmail');
+const errorPassword = document.getElementById('errorPassword');
+const errorPasswordRepeat = document.getElementById('errorPasswordRepeat');
+
+const ctaSignUpBtn = document.getElementById('ctaSignUp');
+const eyeButtons = document.getElementsByClassName('eye-button');
+const eyeOffs = document.getElementsByClassName('eyeOff');
+
+const EMAIL_CHECK = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+const PASSWORD_CHECK = /^(?=.*?[a-z])(?=.*?[0-9]).{8,}$/
+const TEST_EMAIL = 'test@codeit.com'
+const TEST_PASSWORD = 'codeit101'
+
+const emailValidChecker = (email) => {
+    return EMAIL_CHECK.test(email) 
+}
+const passwordValidChecker = (pw) => {
+    return PASSWORD_CHECK.test(pw) 
+}
+
+const addWarningMessage = function (message, messageType) {
+    messageType.textContent = message;
+    this.classList.add('invalid'); 
+}
+
+emailInput.addEventListener('focusout', () => emailChecker());

```

Comment:
```suggestion
emailInput.addEventListener('focusout', emailChecker);
```
호이스팅에 따라서 emailChecker 가 선언된 이후에 이벤트 리스너로 등록해주세요~